core-web/apps/dotcms-ui/src/app/view/components/_common/dot-dropdown-component/dot-dropdown.component.html

```diff
@@ -16,14 +16,12 @@
     pButton
     icon="pi pi-chevron-down"
     iconPos="right"
-    data-testid="title-button"
-></button>
+    data-testid="title-button"></button>
 
 <div
     class="dropdown-content"
     *ngIf="show"
     [@enterAnimation]="show"
-    [ngStyle]="!position || position === 'left' ? { left: 0 } : { right: 0 }"
->
+    [ngStyle]="!position || position === 'left' ? { left: 0 } : { right: 0 }">
```

Comment:
```suggestion
    [ngStyle]="{ [position]: 0 }">
```

And set position left by default

```ts
 @Input() position: "left" | "right" = "left"; // By default
```